src/sonic-build-hooks/hooks/apt-get

```diff
@@ -20,11 +26,29 @@ if [ "$INSTALL" == y ]; then
     [ "$lock_result" == y ] && release_apt_installation_lock
     exit $command_result
 else
-    if [[ "$1" == "purge" || "$@" == *" purge "* ]]; then
+    if [[ "$1" == "update" || "$@" == *" update "* ]]; then
+        set -x
+        if [[ "${FORCE_UPDATE}" == Y  ]]; then
+            UPDATE=y
+        elif $(set -- /var/lib/apt/lists/*_debian_dists_${DISTRO}_InRelease; test -e "$1"); then
```

Comment:
"$1" is "update"?

Reply:
Yes, $1 is an update arg which is coming from 'apt-get update'. During apt-get update, it will cache the debian package release files. so that subsequent build will be loaded from cache.